### PR TITLE
4212 silas rslaser update

### PR DESCRIPTION
**Pull request by gurhar1133**
Changes made to silas to correctly instantiate LaserCavity given recent rslaser changes

### CODE REVIEW

**Review by e-carlin — COMMENTED** [review]
LGTM, couple nits

File: sirepo/package_data/template/silas/parameters.py.jinja [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 from pykern.pkcollections import PKDict
-from rslaser import cavity
-from scipy import ndimage, constants
+from rslaser.cavity import laser_cavity
+from scipy import ndimage
 import array
 import h5py
 import math
 import numpy as np
+import scipy.constants as const
```
[the remark this answers is not in the document]
Comment (gurhar1133) [reply sirepo/package_data/template/silas/parameters.py.jinja:9]: Ok, import numpy as np is only exception?

**Comment by gurhar1133** [comment]
Changes made.

File: sirepo/package_data/template/silas/parameters.py.jinja (hunk 1 above)
[the remark this answers is not in the document]
Comment (e-carlin) [reply sirepo/package_data/template/silas/parameters.py.jinja]: No you can fix that one too (and others as you come across them).

**Review by e-carlin — APPROVED** [review]

**Pull request merged**